crates/object_detection/src/pose_detection.rs

```diff
@@ -222,6 +222,36 @@ impl PoseDetection {
             human_poses: poses.into(),
         })
     }
+
+    fn should_look_for_referee<T: TimeInterface>(
+        &mut self,
+        hardware_interface: &Arc<T>,
```

Comment:
you are only using the hardware interface to get the time. Do that once and pass the actual `now` to the function. You should maybe look into using `CycleInfo` instead anyway.

Reply:
Done